lambdas/process-cri-callback/src/test/java/uk/gov/di/ipv/core/processcricallback/pact/ninoCri/ContractTest.java

```diff
@@ -0,0 +1,667 @@
+                   "value": "1965-07-08"
+                 }
+               ]
+             },
+             "type": [
+               "VerifiableCredential",
+               "IdentityCheckCredential"
+             ]
+           },
+           "jti":"dummyJti"
+         }
+    """;
+    // If we generate the signature in code it will be different each time, so we need to generate a
+    // valid signature (using https://jwt.io works well) and record it here so the PACT file doesn't
+    // change each time we run the tests.
+    private static final String VALID_NINO_VC_SIGNATURE =
+            "NbGQB9HuVOhRS-GCmJR3oHGRGpZJFuUq8omDdP2odrBmFeqDTwd2NAo0v4k6yl9WLYJtW7XmO1eufdLPy-5Vbg";
+
+    private static final String FAILED_NINO_VC_BODY =
+            """
+            {
+                "sub": "test-subject",
+                "iss": "dummyNinoComponentId",
+                "nbf": 4070908800,
+                "exp": 4070909400,
+                "vc": {
+                 "evidence": [
+                   {
+                     "failedCheckDetails": [
+                       {
+                         "checkMethod": "data"
+                       }
+                     ],
+                     "validityScore": 0,
+                     "strengthScore": 2,
+                     "ci": [
+                        "D02"
+                     ],
+                     "txn": "dummyTxn",
+                     "type": "identityCheck"
+                   }
+                 ],
+                 "credentialSubject": {
+                   "socialSecurityRecord": [
+                     {
+                       "personalNumber": "AA000003D"
+                     }
+                   ],
+                   "name": [
+                     {
+                       "nameParts": [
+                         {
+                           "type": "GivenName",
+                           "value": "Kenneth"
+                         },
+                         {
+                           "type": "FamilyName",
+                           "value": "Decerqueira"
+                         }
+                       ]
+                     }
+                   ],
+                   "birthDate": [
+                     {
+                       "value": "1965-07-08"
+                     }
+                   ]
+                 },
+                 "type": [
+                   "VerifiableCredential",
+                   "IdentityCheckCredential"
+                 ]
+                },
+                "jti":"dummyJti"
+                }
+            """;
+    // If we generate the signature in code it will be different each time, so we need to generate a
+    // valid signature (using https://jwt.io works well) and record it here so the PACT file doesn't
+    // change each time we run the tests.
+    private static final String FAILED_NINO_VC_SIGNATURE =
+            "KMaVibLlS-9rsnH26-M6TV7dlmUMRNISyHD5e0nZHiIWryFLQ67buXVRhxZ-_ZW64TAeVouvmNccmVb1716QfA";
+
```

Comment:
Not sure if it's out of date, but this Confluence page has a different type of failure VC. If it's correct then we should have a test for this too.
https://govukverify.atlassian.net/wiki/spaces/Architecture/pages/3819930131/NINO+record+check+Technical+specification#Failed-NINO-check

Reply:
Actually the data in the stub CRI matches Confluence and not the VCs in here (stub data has no strength score etc) where did these VCs come from?

Reply:
I pulled this from an actual test journey that I ran in orch-stub so this is exactly what we are getting back from the stub

Reply:
Hm but that's different to the VC that is stored in DynamoDb, so maybe core-front is concatenating something when it displays it to the user.

Not really sure how that's possible though

Reply:
So it's because we sometimes use the NINO CRI with the identity_check scope, in which case we provide the scores, otherwise we don't.

I'll add some extra tests to cover it.